Project.toml

```diff
@@ -1,5 +1,5 @@
-name = "LatentGPs"
-uuid = "45b822f7-b0f9-4a40-9473-dffa0412f8e7"
+name = "NonConjugateGPs"
+uuid = "6031954c-0455-49d7-b3b9-3e1c99afaf40"
```

Comment:
I guess we could keep the same UUID?

Reply:
I think the UUIDs are tied to the name deterministically. 

https://discourse.julialang.org/t/problem-with-reassignment-of-uuid-when-a-package-is-registered/11422/2
This is an old post. Not sure how true it is.

Reply:
AFAIK this only matters for registered packages (https://github.com/JuliaRegistries/General#how-do-i-rename-an-existing-registered-package).

Reply:
And this package is not registered yet?

Reply:
I didn't realize that the package hasn't been registered yet. 😅 In that case, the change in the UUID shouldn't matter? Am I missing something?

Reply:
True :smile: